PowerEditor/src/TinyXml/tinyxml.cpp

```diff
@@ -555,9 +555,9 @@ const char one_ws[] = " ";
 const char four_ws[] = "    ";
 const char eol[] = "\r\n";
 
-const generic_string tagOpenSymb = TEXT("<");
-const generic_string tagOpenSlash = TEXT("</");
-const generic_string tagCloseSymb = TEXT(">");
+const char tagOpenSymb[] = "<";
+const char tagOpenSlash[] = "</";
+const char tagCloseSymb[] = ">";
```

Comment:
If you add `write(const std::string& str)` method as suggested above maybe it will be a good idea to make all `const char` values here to be `const std::string`. This will remove the `strlen()` call when writing them to the file from every place where that happens (and it seems it will happen a lot).

Reply:
It'll be even better to remove them.
For the sake of readability like @mere-human has mentioned, I can reintegrate these strings into code thanks to `writeStr(const std::string& str)`.